server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/SharepointItemSynchronization.cs

```diff
@@ -103,6 +136,8 @@ private async Task UpsertStorageItem(string eventId, string calendar, DatesPerio
                     calendar,
                     upsertItem,
                     cancellationToken);
+                // TODO: blocked for event filtration
```

Comment:
what do you mean? Again, you should not send ANY notification using sharepoint

Reply:
I mean why is even TODO. Why sharepoint has anything to do with notifications at ALL?

Reply:
Removed